pkg/api/v1/types_swagger_doc.go

```diff
@@ -0,0 +1,1164 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+// This file contains a collection of methods that can be used from go-resful to
+// generate Swagger API documentation for its models. Please read this PR for more
+// information on the implementation: https://github.com/emicklei/go-restful/pull/215
+//
+// TODOs are ignored from the parser. (e.g. TODO(andronat):... || TODO:...) iff
+// are on one line! For multiple line or blocks that you want to ignore use ---.
+// Any context after a --- is ignored.
+//
+// Those methods can be generated by using hack/update-generated-swagger-docs.sh
+
+// AUTO-GENERATED FUNCTIONS START HERE
+func (APIVersion) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":         "An APIVersion represents a single concrete version of an object model.",
+		"name":     "Name of this version (e.g. 'v1').",
+		"apiGroup": "The API group to add this object into. Default 'experimental'.",
+	}
+}
+
+func (AWSElasticBlockStoreVolumeSource) SwaggerDoc() map[string]string {
+	return map[string]string{
+		"":          "AWSElasticBlockStoreVolumeSource represents a Persistent Disk resource in AWS.\n\nAn Amazon Elastic Block Store (EBS) must already be created, formatted, and reside in the same AWS zone as the kubelet before it can be mounted. Note: Amazon EBS volumes can be mounted to only one instance at a time.",
```

Comment:
Sorry i missed this. We want to avoid repeating the names of the func or properties. Also we should use lower case "persistent disk" here (its just describing the type of resource and is not a proper noun). Can we please remove/change to:

"~~AWSElasticBlockStoreVolumeSource~~ Represents a persistent disk resource in AWS.   ..."